nimrod/test_suite_generation/generators/test_suite_generator.py

```diff
@@ -73,6 +74,7 @@ def _compile_test_suite(self, input_jar: str, test_suite_path: str, extra_class_
             try:
                 self._java.exec_javac(java_file, test_suite_path, None, None,
                                     '-classpath', class_path, '-d', compiled_classes_path)
-            except:
-                pass
+            except CalledProcessError:
+                logging.error(f"Error while compiling {java_file}")
+                continue
```

Comment:
Eu apenas removeria o `continue`, considerando que a exceção é capturada e tratada, o `for` continua naturalmente com a sua próxima interação.

Reply:
Removido!